js/game.js

```diff
@@ -206,92 +187,67 @@ Game.prototype.build = function(xOnBoard,yOnBoard) {
     this.selectedBuilding = undefined;
     View.deselectBuilding();
   }
-}
+};
 
 Game.prototype.buildCurrentBuildOrder = function() {
-  var building = this.currentBuildOrder
-  if (building) {
-    if (building.completed) {
-      building = undefined;
-    } else {
-      building.continueBuilding();
-      this.resources.matter -= building.matterToDeductPerCycle();
-      this.resources.energy -= building.energyToDeductPerCycle();
-    }
+  var building = this.currentBuildOrder;
+  if (building && !building.completed) {
+    building.continueBuilding();
+    this.resources.matter -= building.matterToDeductPerCycle();
+    this.resources.energy -= building.energyToDeductPerCycle();
   }
-}
+};
 
-Game.prototype.areBuildingsDestroyed = function() {
-  var isAtLeastOneDestroyed = false
+Game.prototype.fireBuildingsAtEnemies = function() {
   for (var i = 0; i < this.buildings.length; i++) {
-    if (this.buildings[i].isDestroyed()) {
-      isAtLeastOneDestroyed = true;
-      if (!this.buildings[i].completed) {
-        this.currentBuildOrder = undefined;
```

Comment:
In the original function, this is a side effect that's unexpected from reading the function's name. I just removed it for now since it doesn't seem to break anything. Thoughts?